packages/link/src/__demo__/examples/link.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Link from '../../Link';
+
+export default {
+  title: 'Basic link',
+  description: 'Links can be used in prose to create hypertext.',
+  scope: { Link },
+  source: `<p>
+Sensors indicate human life forms 30 meters below the planet surface. Stellar flares are increasing in magnitude and frequency. Set course for <Link target="_blank" href="http://memory-alpha.wikia.com/wiki/Rhomboid_Dronegar_sector_006">Rhomboid Dronegar 006</Link>, warp seven. There is no evidence of an advanced communication network. Total guidance system failure, with less than 24 hours reserve power. Shield effectiveness has been reduced 12 percent. We have covered the area in a spherical pattern which a ship without warp drive could cross in the given time.
```

Comment:
Please just reuse the lipsum that is used elsewhere on the site.  We want to try to keep the language standard/professional until we have defined a voice of our own.